ReleaseHistory.md

```diff
@@ -18,6 +18,8 @@
 ## UNRELEASED
 * DEP: Remove `Microsoft.CodeAnalysis`. [#934](https://github.com/microsoft/binskim/pull/934)
 * DEP: Remove `Microsoft.CodeAnalysis.NetAnalyzers`. [#934](https://github.com/microsoft/binskim/pull/934)
+* DEP: Update `msdia140.dll` from 14.32.31326.0 to 14.36.32532.0. [936](https://github.com/microsoft/binskim/pull/936)
+* DEP: Update `symsrv.dll` from 10.0.10150.0 to 10.0.22621.755. [936](https://github.com/microsoft/binskim/pull/936)
```

Comment:
>symsrv.dll [](http://example.com/codeflow?start=15&length=10)

should be good to update this even does not resolve bugs, it is used for symbol server.